charts/edge-caiasoft/values.yaml

```diff
@@ -81,9 +81,7 @@ jmx:
 
 heapDumpEnabled: false
 
-eureka:
-  enabled: false
-  sidecarContainer: {}
+eureka: &is_eureka false
```

Comment:
Please adjust logic in folio-common

Reply:
PR has been split into two separate PRs for convenience reasons. Please take a look at https://github.com/folio-org/folio-helm-v2/pull/163.